.doc_gen/metadata/bedrock-runtime_metadata.yaml

```diff
@@ -206,6 +206,15 @@ bedrock-runtime_Scenario_ToolUse:
               genai: some
               snippet_tags:
                 - Bedrock.ConverseTool.dotnetv3.SendConverseRequest
+    JavaScript:
```

Comment:
There should be 3 keys updated in the metadata, refer to the table in the specification, looks like 2 are missing.